Application/mobile/app_code/lib/pages/contact/conversation_state.dart

```diff
@@ -0,0 +1,222 @@
+import 'package:provider/provider.dart';
+import 'package:risu/components/appbar.dart';
+import 'package:risu/components/loader.dart';
+import 'package:risu/globals.dart';
+import 'package:risu/utils/errors.dart';
+import 'package:risu/utils/providers/theme.dart';
+import 'package:risu/utils/time.dart';
+
+import 'conversation_page.dart';
+
+class ConversationPageState extends State<ConversationPage> {
+  List<dynamic> tickets = [];
+  bool isOpen = false;
+
+  @override
+  void initState() {
+    super.initState();
+    tickets = widget.tickets;
+    isOpen = widget.isOpen;
+  }
+
+  final LoaderManager _loaderManager = LoaderManager();
+
+  Future<bool> postTicket(String content) async {
+    late http.Response response;
+    dynamic previousTicket = tickets.last;
+
+    try {
+      setState(() {
+        _loaderManager.setIsLoading(true);
+      });
+      response = await http.post(Uri.parse('$baseUrl/api/mobile/ticket/'),
+          headers: <String, String>{
+            'Content-Type': 'application/json; charset=UTF-8',
+            'Authorization': 'Bearer ${userInformation?.token}',
+          },
+          body: jsonEncode(<String, String>{
+            'content': content,
+            'title': previousTicket["title"],
+            'assignedId': previousTicket["assignedId"],
+            'chatUid': previousTicket["chatUid"],
+          }));
+      setState(() {
+        _loaderManager.setIsLoading(false);
+      });
+      if (response.statusCode == 201) {
+        return true;
+      } else {
+        if (context.mounted) {
+          printServerResponse(context, response, 'postTicket',
+              message:
+                  AppLocalizations.of(context)!.errorOccurredDuringPostTicket);
+          return false;
+        }
+      }
+    } catch (err, stacktrace) {
+      if (context.mounted) {
+        setState(() {
+          _loaderManager.setIsLoading(false);
+        });
+        printCatchError(context, err, stacktrace,
+            message:
+                AppLocalizations.of(context)!.errorOccurredDuringPostTicket);
+        return false;
+      }
+    }
+    return true;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final themeProvider = Provider.of<ThemeProvider>(context);
+    final scrollController = ScrollController();
+    final contentController = TextEditingController();
+
+    return Scaffold(
+      appBar: MyAppBar(
+        curveColor: context.select((ThemeProvider themeProvider) =>
+            themeProvider.currentTheme.secondaryHeaderColor),
+        showBackButton: false,
+        showLogo: true,
+      ),
+      resizeToAvoidBottomInset: true,
+      backgroundColor: context.select((ThemeProvider themeProvider) =>
+          themeProvider.currentTheme.colorScheme.background),
+      body: Container(
+        margin: const EdgeInsets.symmetric(horizontal: 16, vertical: 32),
+        child: Column(
+          children: [
+            Text(
+              tickets[0]["title"],
```

Comment:
Why tickets[0] ?

Reply:
Because it's the first ticket of the list, so it will always exist and it will hold the title, although every ticket of the same chat have the same